src/main/bin/ccm-fetch

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/perl ## no critic ((RequireVersionVar);
```

Comment:
That makes me even less keen on it - it should only be needed temporarily and no one will be brave enough to  remove it!